guide/english/css/graceful-degradation/index.md

```diff
@@ -17,4 +17,18 @@ title: Graceful Degradation
  ### Example
  
  A simple example is the use of 24-bit alpha-transparent PNGs. Those images can be displayed on modern browsers without problems. IE5.5 and IE6 would show the image, but transparency effects would fail (it can be made to work if necessary). Older browsers that do not support PNG would show alt text or an empty space.
- 
+
+
+### Resources
+
+[How can polyfills be used as a fallback for older browsers](https://developer.mozilla.org/en-US/docs/Glossary/Polyfill)
+
+[What is polyfill](https://remysharp.com/2010/10/08/what-is-a-polyfill)
+
+[Online HTML beautifier](https://www.10bestdesign.com/dirtymarkup/)
+
+[W3C CSS validator](http://jigsaw.w3.org/css-validator/)
+
+[Online CSS linter](http://csslint.net/)
```

Comment:
This website is not supporting `https`, which is not conform with the [freeCodeCamp's Style Guide](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/docs/style-guide-for-guide-articles.md#links). Please remove it from your changes.